elasticdl/python/elasticdl/feature_column/feature_column.py

```diff
@@ -132,6 +133,16 @@ def variable_shape(self):
         """See `DenseColumn` base class."""
         return tensor_shape.TensorShape([self.dimension])
 
+    def create_state(self, state_manager):
+        # Get the name of the embedding variable
+        EMBEDDING_VARIABLE_NAME_FORMAT = "{}/{}/embedding_weights:0"
```

Comment:
"{dense_feature_layer_name}/{embedding_column_name}/embedding_weights:0" may be more readable.

Reply:
Good idea, updated.